org.hydrogenmusic.hydrogen.json

```diff
@@ -0,0 +1,174 @@
+{
+  "app-id": "org.hydrogenmusic.hydrogen",
+  "branch": "stable",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.11",
+  "sdk": "org.kde.Sdk",
+  "command": "hydrogen",
+  "rename-desktop-file": "hydrogen.desktop",
+  "rename-appdata-file": "hydrogen.appdata.xml",
+  "rename-icon": "h2-icon",
+  "finish-args": [
+    /* X11 + XShm access */
+    "--share=ipc",
+    "--socket=x11",
+    /* Wayland */
+    "--socket=wayland",
+    /* Note playback */
+    "--socket=pulseaudio",
+    /* MIDI */
+    "--device=all",
+    /* Allow loading, saving files from anywhere (portals don’t work yet) */
+    "--filesystem=host",
+    /* Allow other instances to see lockfiles */
+    "--env=TMPDIR=/var/tmp",
+    "--env=QT_AUTO_SCREEN_SCALE_FACTOR=1"
+  ],
+  "modules": [
```

Comment:
You should add global or per module cleanup section to remove development headers (and so on).

Reply:
done